### PR TITLE
#3586-UI-CV-test_positive_publish_version_changes_in_target-source_env

### PR DESCRIPTION
**Pull request by ldjebran**
Issue https://github.com/SatelliteQE/robottelo/issues/3586

### CODE REVIEW

**Comment by ldjebran** [comment]
```bash
(2.7) dlezz@elysion:~/projects/robottelo-fork$ py.test tests/foreman/ui/test_contentview.py -v -k 'test_positive_publish_version_changes_in_target_env'
================================================= test session starts ==================================================
platform linux2 -- Python 2.7.12, pytest-3.0.3, py-1.4.31, pluggy-0.4.0 -- /home/dlezz/bin/redhat/python/2.7/bin/python
cachedir: .cache
rootdir: /home/dlezz/projects/robottelo-fork, inifile: 
plugins: cov-2.4.0, xdist-1.15.0
collected 64 items 

tests/foreman/ui/test_contentview.py::ContentViewTestCase::test_positive_publish_version_changes_in_target_env <- robottelo/decorators/__init__.py PASSED

================================================= 63 tests deselected ==================================================
====================================== 1 passed, 63 deselected in 374.92 seconds =======================================
```

**Comment by ldjebran** [comment]
```bash
(2.7) dlezz@elysion:~/projects/robottelo-fork$ py.test tests/foreman/ui/test_contentview.py -v -k 'test_positive_publish_version_changes_in_source_env'
================================================= test session starts ==================================================
platform linux2 -- Python 2.7.12, pytest-3.0.3, py-1.4.31, pluggy-0.4.0 -- /home/dlezz/bin/redhat/python/2.7/bin/python
cachedir: .cache
rootdir: /home/dlezz/projects/robottelo-fork, inifile: 
plugins: cov-2.4.0, xdist-1.15.0
collected 64 items 

tests/foreman/ui/test_contentview.py::ContentViewTestCase::test_positive_publish_version_changes_in_source_env <- robottelo/decorators/__init__.py PASSED

================================================= 63 tests deselected ==================================================
====================================== 1 passed, 63 deselected in 379.09 seconds =======================================
```

File: tests/foreman/ui/test_contentview.py [hunk 1]
```diff
@@ -115,6 +115,19 @@ def setup_to_create_cv(self, repo_name=None, repo_url=None, repo_type=None,
...
 
+    def _get_cv_version_environments(self, cv_version):
+        """
+        Return the list of environments promoted to the version of content
+        view.
+        The content view web page must be already opened
+        :param cv_version: The version of the current opened content view
+        :type cv_version: str
+        :rtype: list[str]
+        """
```
Comment (abalakh) [inline tests/foreman/ui/test_contentview.py]:
```python
        """Return the list of environments promoted to the version of content
        view. The content view web page must be already opened.

        :param cv_version: The version of the current opened content view
        :type cv_version: str
        :rtype: list[str]

        """
```
Tip: you can use any online rst editor to check if your docstring is ok, for example [rst.ninjs.org](http://rst.ninjs.org/)

**Comment by renzon** [comment]
ACK, can merge after @abalakh observation

**Pull request merged**